### DISCUSSION TITLE
Get item form values inside Repeater

### DISCUSSION THREAD

**Discussion by enzolupia** [directus/directus#13436] [Archive]
Hi everyone, from what I see, the Directus v-form component use the _provide_ directive to provide the configured values to child component.
By injecting the _values_ property I can easily get the values of a new item (while editing) and use them inside a new custom interface we implemented.
This work great, but I'm looking for a way to retrieve the item values from inside a Repeater form.
Currently we have a repeater with a single field (the custom interface), whose content must change according to the value set to another field in the same collection.

* Collection
  * field1
  * field2
  * repeater
    * custom interface field (contains a dropdown whose options must change according to field1)

The injected _values_ inside the custom interface obviously allows me to get the values of the repeater itself, but how can I get the values of the outer form?

Thanks

**Comment by rijkvanzanten** [comment]
There's currently no way to retrieve the values a level up. The form in the repeater is a new standalone form context, that doesn't have direct connection to the parent level form at the moment.
